### PR TITLE
Fix : Documentation of examples

### PR DESCRIPTION
**Pull request by dipinknair**

### CODE REVIEW

File: doc/Makefile [hunk 1]
```diff
@@ -26,6 +26,7 @@ help:
 clean:
 	rm -rf $(BUILDDIR)/*
 	rm -rf $(SOURCEDIR)/examples
+	rm -rf source/examples/gallery_examples
```
Comment (RobPasMue) [inline doc/Makefile:29]:
```suggestion
	rm -rf $(SOURCEDIR)/examples/gallery_examples
```

Also, why is this duplicated?

File: doc/source/conf.py [hunk 2]
```diff
@@ -59,9 +60,9 @@
     # convert rst to md for ipynb
     "pypandoc": True,
     # path to your examples scripts
-    "examples_dirs": ["../../examples/basic"],
+    "examples_dirs": ["../../examples/"],
     # path where to save gallery generated examples
-    "gallery_dirs": ["basic"],
+    "gallery_dirs": ["examples/gallery_examples"],
```
Comment (RobPasMue) [inline doc/source/conf.py:65]: I would leave it as examples if possible...

**Review by RobPasMue — APPROVED** [review]
LGTM - some minor comments and suggestions

File: doc/Makefile (hunk 1 above)
Comment (dipinknair) [reply doc/Makefile]: Fixed as per suggestions.

File: doc/source/conf.py (hunk 2 above)
Comment (dipinknair) [reply doc/source/conf.py]: Updated "examples/gallery_examples" -> "examples"

**Comment by dipinknair** [comment]
Close #63 & #71

**Pull request merged**